video/repack.c

```diff
@@ -214,7 +214,7 @@ static void swap_endian(struct mp_image *dst, int dst_x, int dst_y,
 
 
 #define PA_WORD_3(name, packed_t, plane_t, sh_c0, sh_c1, sh_c2, pad)        \
-    static void name(void *dst, void *src[], int w) {                       \
+    static void name(void *restrict dst, void *restrict src[], int w) {     \
```

Comment:
function signature could be turned into a macro to deduplicate it

Reply:
not sure it would make it clearer, there are few different signatures and file pointers. Would make more mess I think.